### PR TITLE
Improve Insertion Titles

### PR DESCRIPTION
**Pull request by dpoeschl**
Include the originating branch name & build number

### CODE REVIEW

**Comment by dpoeschl** [comment]
So, I haven't actually tested this. Need to figure out how to build & run on my machine. Anyway, tagging @brettfo for an early review.

File: src/RoslynInsertionTool/RoslynInsertionTool/RoslynInsertionTool.VisualStudioTeamServices.cs [hunk 1]
```diff
@@ -44,7 +44,7 @@ private static GitPullRequest CreatePullRequest(string sourceBranch, string targ
             Log.Trace($"Creating pull request sourceBranch:{sourceBranch} targetBranch:{targetBranch} description:{description}");
             return new GitPullRequest
             {
-                Title = $"{Options.InsertionName} Insertion Into {Options.VisualStudioBranchName} ",
+                Title = $"{Options.InsertionName} '{Options.BranchName}/{Options.SpecificBuild}' Insertion into {Options.VisualStudioBranchName} ",
```
Comment (brettfo) [inline src/RoslynInsertionTool/RoslynInsertionTool/RoslynInsertionTool.VisualStudioTeamServices.cs]: You'll want to pass through and consume `buildToInsert` from [here](https://github.com/dotnet/roslyn-tools/blob/e22a64f07f5c84b261ce57c10ef29f2757d97a79/src/RoslynInsertionTool/RoslynInsertionTool/RoslynInsertionTool.cs#L110) (and also line 100 above) (or `buildVersion`).  `Options.SpecificBuild` is a command-line override that will only be used in manual cases.

**Comment by dpoeschl** [comment]
FYI @jamshedd that this is being worked on.

**Comment by dpoeschl** [comment]
@brettfo I've updated it with your suggestions. Can you give it another look?

**Pull request merged**